packages/block-library/src/image/edit.js

```diff
@@ -8,6 +8,7 @@ import {
 	map,
 	last,
 	pick,
+	round,
```

Comment:
why did we bring in `lodash.round` instead of using `Math.round()` the native function?

Reply:
Uh, that was used in previous iterations to round to specific precision. Not used any more so can remove it :)

Reply:
> Uh, that was used in previous iterations to round to specific precision. Not used any more so can remove it :)

Changed to `Math.round` in 32d5c8c